cross/x264/Makefile

```diff
@@ -6,15 +6,41 @@ PKG_DIST_SITE = ftp://ftp.videolan.org/pub/x264/snapshots/
 PKG_DIR = $(PKG_NAME)-snapshot-$(PKG_VERS)
 
 DEPENDS =
-GNU_CONFIGURE = 1
+
 HOMEPAGE = http://www.videolan.org/developers/x264.html
 COMMENT  = x264 is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC forma
 LICENSE  =
 
-CONFIGURE_ARGS = --enable-static --disable-asm --prefix=$(INSTALL_PREFIX)
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --enable-static --prefix=$(INSTALL_PREFIX)
 
-ifeq ($(findstring $(ARCH), 88f6281),$(ARCH))
+include ../../mk/spksrc.cross-cc.mk
+
+
+ifeq ($(findstring $(ARCH), $(ARM5_ARCHES)),$(ARCH))
```

Comment:
This should be moved to before the `include`...but I guess you ran into issues where `ARM5_ARCHES` wasn't recognized?